src/drive/mobile/lib/cozy-helper.js

```diff
@@ -31,8 +33,50 @@ export const getOauthOptions = () => {
   }
 }
 
-export const initClient = url => {
+export const getOldAdapterName = () => {
+  return isMobileApp() && isIOSApp() ? 'cordova-sqlite' : 'idb'
+}
+
+export const getAdapterPlugin = adapterName => {
+  if (adapterName === 'cordova-sqlite') {
+    return require('pouchdb-adapter-cordova-sqlite')
+  }
+  if (adapterName === 'idb') {
+    return require('pouchdb-adapter-idb')
+  }
+  return require('pouchdb-adapter-indexeddb').default
+}
+
+export const shouldMigrateAdapter = async () => {
+  const alreadyOnboarded = await localForage.getItem(ONBOARDED_ITEM)
+  const adapterName = window.localStorage.getItem(
+    'cozy-client-pouch-link-adaptername'
+  )
```

Comment:
nit: Maybe we should use a method from Cozyclient to get the item. This is not the responsability of the app to know the key

Reply:
Right, I assigned an issue to the guy who did that: https://github.com/cozy/cozy-client/issues/1005